### PR TITLE
Add missing version script for autocomplete client

### PR DESCRIPTION
**Pull request by bidoubiwa**
The version script required by the changeset CI to update the versions of the package was missing.

### CODE REVIEW

File: packages/autocomplete-client/scripts/update_version.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const fs = require('fs')
+const path = require('path')
+
+const version = pkg.version
+const fileContents = `export const PACKAGE_VERSION = '${version}'
+`
+
+const filePath = path.resolve(__dirname, '../src/package-version.ts')
+
+console.log(filePath)
```
Comment (alallema) [inline packages/autocomplete-client/scripts/update_version.js:11]: If it's a debug log, maybe a sentence will be great. If not it should be remove

File: packages/autocomplete-client/scripts/build.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// Definitions by: ${author} <https://github.com/${githubUserUrl}>
+// Definitions: ${repoUrl}
+// TypeScript Version: ${tsVersion}
+`.replace(/^\s+/gm, '')
+}
+
+/**
+ *
+ * @param {string} path
+ * @param {string | Blob} data
```
Comment (alallema) [inline packages/autocomplete-client/scripts/build.js:54]: What is `Blob`?

**Review by alallema — COMMENTED** [review]
I don't understand the purpose of this script. It said to build and update the version for the second, but it seems to write some files.
Sorry, js isn't really my strong suit.

File: packages/autocomplete-client/scripts/update_version.js (hunk 1 above)
Comment (bidoubiwa) [reply packages/autocomplete-client/scripts/update_version.js:11]: I'm removing it

File: packages/autocomplete-client/scripts/build.js (hunk 2 above)
Comment (bidoubiwa) [reply packages/autocomplete-client/scripts/build.js:54]: Here you go: https://developer.mozilla.org/en-US/docs/Web/API/Blob

**Review by alallema — APPROVED** [review]
LGTM!

**Pull request merged**